packages/astro/src/@types/astro.ts

```diff
@@ -127,22 +128,28 @@ export interface AstroUserConfig {
 
 	/**
 	 * @docs
-	 * @name renderers
-	 * @type {string[]}
-	 * @default `['@astrojs/renderer-svelte','@astrojs/renderer-vue','@astrojs/renderer-react','@astrojs/renderer-preact']`
+	 * @name integrations
+	 * @type {AstroIntegration[]}
+	 * @default `["react", "preact", "svelte", "vue"]`
 	 * @description
-	 * Set the UI framework renderers for your project. Framework renderers are what power Astro's ability to use other frameworks inside of your project, like React, Svelte, and Vue.
+	 * Add Integrations to your project to extend Astro.
+	 *
+	 * Integrations are your one-stop shop to add new frameworks (like Solid.js), new features (like sitemaps), and new libraries (like Partytown and Turbolinks).
+	 *
+	 * Setting this configuration will disable Astro's default integration, so it is recommended to provide a renderer for every framework that you use:
 	 *
-	 * Setting this configuration will disable Astro's default framework support, so you will need to provide a renderer for every framework that you want to use.
+	 * Note: Integrations are currently under active development, and only first-party integrations are supported. In the future, 3rd-party integrations will be allowed.
 	 *
 	 * ```js
+	 * import react from '@astrojs/react';
+	 * import vue from '@astrojs/vue';
 	 * {
-	 *   // Use Astro + React, with no other frameworks.
-	 *   renderers: ['@astrojs/renderer-react']
+	 *   // Example: Use Astro with Vue + React, and no other frameworks.
+	 *   renderers: [react(), vue()]
 	 * }
 	 * ```
 	 */
-	renderers?: string[];
```

Comment:
ah right, I forgot about TS. Right now we do give a really helpful warning when you run it with `renderers`, but I agree that we'd rather you hit that first instead of seeing the red squiggly and feeling like you need to do your own research to solve.